### PR TITLE
completed array shift code challenge and wrote 3 tests, updated readm…

### PR DESCRIPTION
**Pull request by SharinaS**
…e and added picture of white board
 - [x] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x ] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/main/java/code401Challenges/ArrayShift.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package code401Challenges;
+
+public class ArrayShift {
+
```
Comment (jeffborda) [inline code401Challenges/src/main/java/code401Challenges/ArrayShift.java]: You're correct! the main difference between libraries and applications is that libraries don't have a main method.

File: code401Challenges/src/test/java/code401Challenges/ArrayShiftTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package code401Challenges;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
```
Comment (jeffborda) [inline code401Challenges/src/test/java/code401Challenges/ArrayShiftTest.java]: Excellent job thinking about different test cases to consider.
